src/devices/Ft4222/Ft4222Gpio.cs

```diff
@@ -35,49 +35,64 @@ public class Ft4222Gpio : GpioDriver
         /// <summary>
         /// Create a FT4222 GPIO driver
         /// </summary>
-        /// <param name="deviceNumber"></param>
+        /// <param name="deviceNumber">Number of the device in the device list, default 0</param>
```

Comment:
```suggestion
        /// <param name="deviceNumber">Number of the devices in the device list, default 0</param>
```

Reply:
I don't think so, it's an index, not a count.